tests/Form/ChoiceList/ModelChoiceLoaderTest.php

```diff
@@ -17,14 +17,30 @@
 use Sonata\AdminBundle\Form\ChoiceList\ModelChoiceLoader;
 use Sonata\AdminBundle\Model\ModelManagerInterface;
 use Sonata\AdminBundle\Tests\Fixtures\Bundle\Entity\Foo;
+use Symfony\Bridge\PhpUnit\ExpectDeprecationTrait;
 
 class ModelChoiceLoaderTest extends TestCase
 {
+    use ExpectDeprecationTrait;
+
     private $modelManager;
 
     protected function setUp(): void
     {
-        $this->modelManager = $this->getMockForAbstractClass(ModelManagerInterface::class);
+        $this->modelManager = $this->createMock(ModelManagerInterface::class);
+    }
+
+    /**
+     * NEXT_MAJOR: Check for a InvalidArgumentException instead of the deprecation.
+     *
+     * @group legacy
+     */
+    public function testConstructWithUnsupportedQuery(): void
+    {
+        $this->modelManager->method('supportsQuery')->willReturn(false);
```

Comment:
```
Trying to configure method "supportsQuery" which cannot be configured because it does not exist, has not been specified, is final, or is static
```
You should use add the method to the mock

Reply:
How ?

Reply:
```php
$stub = $this
    ->getMockBuilder('SomeClass')
    ->setMethods(['execute'])
    ->getMock();
```

Reply:
Just tried, it doesn't work with an Interface.

I removed the test and will add a test on master after the merge then.

Reply:
I don't know the use case, but have you tried with `getMockForAbstractClass()`?

Reply:
Thanks @phansys, it works with `getMockForAbstractClass`.